qcore/caching.pxd

```diff
@@ -21,7 +21,7 @@ cdef object miss
 cdef object not_computed
 
 
-cdef class LazyConstant(object):
+cdef class LazyConstant:
```

Comment:
I'm not sure whether this makes a difference to Cython. Could you check their docs?

Reply:
I'm looking at the followin docs on cdef-classes and although on non-Cython example has an `(object)` marker, there's no use of it in the rest of the doc, and I couldn't find any implicit indication that it would make a difference.

- https://cython.readthedocs.io/en/latest/src/tutorial/cdef_classes.html
- https://cython.readthedocs.io/en/latest/src/userguide/extension_types.html

Is there any specific behavior change that you would be concerned about? If so, we might need a unit test for that?